metrics/github/cli.py

```diff
@@ -169,6 +169,11 @@ def tech_owned_repo(pr):
     return not (pr["org"] in NON_TECH_REPOS and pr["repo"] in NON_TECH_REPOS[pr["org"]])
 
 
+def fetch_prs(client):
```

Comment:
I was confused by when we use `iter_*` and when we use `fetch_*`, but having read through the PR I don't think there's a convention, as such (both signal generator functions). Have I misunderstood?

Reply:
That's a good spot. We've already used `iter_prs` in prs.py, which is where this function ended up, so I needed a different name. I think there's probably a better name for that function though or some further refactoring needed to stop us from wanting to call two separate functions the same thing.